app/src/main/java/eu/faircode/netguard/Receiver.java

```diff
@@ -224,13 +224,9 @@ public static void upgrade(boolean initialized, Context context) {
                 Log.i(TAG, "Initializing sdk=" + Build.VERSION.SDK_INT);
                 editor.putBoolean("whitelist_wifi", false);
                 editor.putBoolean("whitelist_other", false);
-                if (Build.VERSION.SDK_INT == Build.VERSION_CODES.LOLLIPOP_MR1) {
```

Comment:
This code should stay to enable filtering by default on Android 5.0, but there is a bug here, since LOLLIPOP_MR1 should read LOLLIPOP. People can choose to disable it, but to prevent complains (the blocking all problem) it should be enabled by default (hence the remark "optional").